test/e2e/data/e2e_eks_conf.yaml

```diff
@@ -16,21 +16,20 @@ images:
   - name: gcr.io/k8s-staging-cluster-api/capa-manager:e2e
     loadBehavior: mustLoad
 
-## PLEASE KEEP THESE UP TO DATE WITH THE COMPONENTS
+  ## PLEASE KEEP THESE UP TO DATE WITH THE COMPONENTS
```

Comment:
nit: Indentation of new changes can be taken care (may be extra tabs are present)

Reply:
Yes I tried in both VSCode and Goland, its still coming

Reply:
yes checking